articles/app-service-mobile/app-service-mobile-windows-store-dotnet-get-started-users.md

```diff
@@ -27,6 +27,20 @@ This tutorial is based on the Mobile Apps quickstart. You must first complete th
 ## <a name="register"></a>Register your app for authentication and configure the App Service
 [!INCLUDE [app-service-mobile-register-authentication](../../includes/app-service-mobile-register-authentication.md)]
 
+## <a name="redirecturl"></a>Add your app to the Allowed External Redirect URLs
+
+Secure authentication requires that you define a new URL scheme for your app. This allows the authentication system to redirect back to your app once the authentication process is complete. In this tutorial, we use the URL scheme _appname_ throughout. However, you can use any URL scheme you choose. It should be unique to your mobile application. To enable the redirection on the server side:
```

Comment:
Thanks @conceptdev. This is very valuable information, we can add this in future update.